### DISCUSSION TITLE
misalignment of linenos with code in code-blocks

### DISCUSSION THREAD

**Discussion by mluerig** [pradyunsg/furo#361] [Potential Issue] [answered]
see here: https://www.phenopype.org/docs/templates/gallery/

I am not sure where to start, there error could be with furo, sphinx-design, or sphinx itself - any thoughts? 

potentially relevant: https://stackoverflow.com/questions/45744846/bad-sphinx-linenos-alignment

**Comment by pradyunsg** [comment]
<img width="717" alt="Screenshot 2022-01-21 at 15 33 03" src="https://user-images.githubusercontent.com/3275593/150554593-0cb91b9e-f9f5-42e6-8861-cc9a8b5a367d.png">

Seems pretty aligned to me. Could you share a screenshot?

**Comment by mluerig** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/15648068/150556882-bf83f700-af99-4e1d-b6d2-5863b5d8afdd.png)

**Reply by mluerig** [reply]
interesting, it does change when changing the modes...

**Reply by pradyunsg** [reply]
Rebuild the docs with the current release. 0c4018fa730d12dc2144745b75c33b922b7cf7d4

**Reply by mluerig** [reply]
well, that was easy - thanks
